renovate.json5

```diff
@@ -18,7 +18,8 @@
         "/^fumadocs-core$/",
         "/^fumadocs-mdx$/",
         "/^fumadocs-ui$/"
-      ]
+      ],
+      "enabled": false  // Temporarily disabled due to Zod v4 dependency conflicts. See route06/liam-internal#5606
```

Comment:
https://docs.renovatebot.com/configuration-options/#:~:text=To%20disable%20Renovate%20for%20all%20eslint%20packages